src/features/search/pages/__tests__/reducer.test.ts

```diff
@@ -55,6 +55,21 @@ describe('Search reducer', () => {
       ...parameters,
     })
   })
+  it('should handle INIT_FROM_SEE_MORE - MAX_PRICE', () => {
+    const parameters = {
+      offerCategories: ['CINEMA', 'MUSIQUE'],
+      priceRange: [30, 500],
+    }
+    const action: Action = {
+      type: 'INIT_FROM_SEE_MORE',
+      payload: parameters as Partial<SearchState>,
+    }
+    expect(searchReducer(state, action)).toStrictEqual({
+      ...initialSearchState,
+      offerCategories: ['CINEMA', 'MUSIQUE'],
+      priceRange: [30, 500],
```

Comment:
```suggestion
      priceRange: [30, 300],
```

je pense que ça vaut le coup de garder le clampPrice dans `INIT_FROM_SEE_MORE`

Reply:
OK